src/pods/generate/GenerateUtil.ts

```diff
@@ -13,7 +13,7 @@ import type { ResourcesGenerator } from './ResourcesGenerator';
  */
 export async function addGeneratedResources(identifier: ResourceIdentifier, settings: NodeJS.Dict<string>,
   generator: ResourcesGenerator, store: ResourceStore): Promise<number> {
-  const resources = generator.generate(identifier, settings);
+  const resources = generator.generate(identifier, { podBaseUrl: identifier.path, ...settings });
```

Comment:
Does this mean the registration result page didn't actually render the pod url correctly up til now? 😅 

It might make more sense to also add this field to the data that is being passed to the `createPod` function instead of updating this utility function?
https://github.com/solid/community-server/blob/fe8d579c72bb5b146e638647644252f05e4bd353/src/identity/interaction/email-password/handler/RegistrationHandler.ts#L138

Reply:
> registration result page

That one was fine (and we integration test that); it receives different parameters.

> add this field to the data that is being passed to the `createPod` function

Fair enough; and then fix that in https://github.com/solid/community-server/issues/845